src/Page/Form.purs

```diff
@@ -158,12 +163,10 @@ component =
           value <- getValue currency
           _ <- toCents value
           pure (print value)
+        , HE.onValueChange (HE.input $ UpdateValue field.key <<< Data.Maybe.maybe UserCleared (UserInput <<< Val <<< Cents) <<< parseCentsFromDollarStr)
```

Comment:
This line got kind of gnarly 😅

Reply:
I moved the `InputSource` from `setValue` to here so we can choose a value other than `UserInput` if it makes sense to. Looks like you had the same idea in you `UserCleared` PR @joneshf-cn